code/__HELPERS/_lists.dm

```diff
@@ -358,7 +358,7 @@
 		return
 	inserted_list = inserted_list.Copy()
 
-	for(var/i = 1, i < inserted_list.len, ++i)
+	for(var/i in 1 to inserted_list.len)
```

Comment:
```suggestion
	for(var/i in 1 to inserted_list.len - 1)
```

You're changing behavior.

Reply:
![image](https://user-images.githubusercontent.com/35135081/142280215-4fc31f82-3c51-42e1-bcd6-06e68e31ec73.png)